Packs/FeedCSV/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "CSV Feed",
     "description": "Indicators feed from a CSV file",
     "support": "xsoar",
-    "currentVersion": "1.1.28",
+    "currentVersion": "1.2.0",
```

as this are minor change
```suggestion
    "currentVersion": "1.1.29",
```